pkg/controller/controller.go

```diff
@@ -174,6 +178,9 @@ func StartLocalController(signal *signal, client *kubernetes.Clientset, ptable d
 		default:
 			deleter.DeletePVs()
 			discoverer.DiscoverLocalVolumes()
+			if !nodeTaintRemover.ShouldRemoveTaint() && discoverer.Readyz.Check(nil) == nil {
```

Comment:
refer to https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/1588, why not use a backoff try if taint removal fails?